README.md

```diff
@@ -49,43 +49,49 @@ repository, as indicated:
    rbenv install
    ```
 
-2. In Limber, make the Bundler gem install the dependencies for this project:
+1. In Limber, make the Bundler gem install the dependencies for this project:
```

Comment:
Annoyingly it's reset these numbers all back to 1, can you undo

Reply:
As discussed off of GitHub, we'll stick with 1s as it's an option you can use in Markdown and makes future diffs cleaner.